src/rgw/setup/rgw.py

```diff
@@ -609,6 +577,44 @@ def _update_hax_endpoint_and_create_admin(conf: MappedConf):
                         raise SetupError(status, 'Admin user creation failed on'
                             f' "{Rgw._machine_id}" node, with all data pods - {data_pod_hostnames}')
 
+    @staticmethod
+    def _get_lock(consul_idx: str):
+        """Aquire consul lock."""
```

Comment:
"""Get lock from consul kv"""

Reply:
"""Aquire consul lock."""
Sounds like we are using ACLs from consul. Which is not the case here.

Reply:
Done.